qtdbusextended/dbusextendedabstractinterface.cpp

```diff
@@ -29,6 +29,12 @@
 #include <QtDBus/QDBusPendingReply>
 #include <QtDBus/QDBusVariant>
 
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
+#define VARIANT_TYPE(metaProperty) QMetaType(metaProperty.userType())
```

Comment:
Maybe `PROPERTY_TYPE`? Or `qMetaPropertyType`?

Reply:
Dunno about name, but would metaProperty.metaType() work as implementation?

Reply:
> Dunno about name, but would metaProperty.metaType() work as implementation?

Yes, that definitely would. I think I dismissed it originally because it was introduced in Qt 6, but now with ifdefs that doesn't matter. Thanks! :slightly_smiling_face:

Reply:
I changed it to that `metaPropertyType` which you suggested.